dbms.py

```diff
@@ -109,3 +121,38 @@ def isBCNF(relation):
         return False
 
     return every(relation.fdSet.leftSideSet(), lambda lhs: relation.validKey(lhs) )
+
+def minimalCover(attributes, fdString):
```

Comment:
`attributes` is never being used here, why not use only the `FunctionalDependencySet` object as argument instead of String?
```suggestion
def minimalCover(fd):
```